core/config/initializers/05_scheduled_jobs.rb

```diff
@@ -1,85 +1,87 @@
 unless Workarea.config.skip_service_connections
-  Sidekiq::Cron::Job.create(
-    name: 'Workarea::CleanInventoryTransactions',
-    klass: 'Workarea::CleanInventoryTransactions',
-    cron: "0 5 * * * #{Time.zone.tzinfo.identifier}",
-    queue: 'low'
-  )
+  Sidekiq.logger.log_at(:error) do
```

Comment:
Sidekiq logs to STDOUT now, so without this you get spammed by these creations at every boot.